main.js

```diff
@@ -0,0 +1,52 @@
+var board = 
+[
+  ['5','4','3','2','1'],
+  [],
+  []
+];
+
+var winningPeg = ['5','4','3','2','1'];
+
+var printBoard = function(board) {
+  var boardAsStringArray = board.map(peg => {
+    return '--- ' + peg.join(' ');
+  });
+  
+  var boardAsString = boardAsStringArray.reduce(function(boardString, item) {
+    return boardString + item + '\n';
+  },'')
+
+  console.log(boardAsString);
+};
+
+var moveDisc = function(pegFrom, pegTo) {
```

Comment:
I'd expect `moveDisc`, `printBoard`, etc to all be part of an object as shown in the prompt's examples. Other than that, this is excellent code. Nice work!